torch/lib/TensorLib/common_with_cwrap.py

```diff
@@ -0,0 +1,111 @@
+# this code should be common among cwrap and TensorLib preprocessing
+# for now, I have put it in one place but right now is copied out of cwrap
```

Comment:
@zdevito @apaszke I have done a proof-of-concept of this and committed to the repo - we load the `common_with_cwrap` module into `cwrap.py` using the existing nnwrap utility, and reuse the function there. If this is acceptable I will migrate the remaining use cases.